Sakura/Modules/messages.py

```
@@ -125,47 +125,27 @@
-    "I can't respond properly right now 😪",
-    "There's some internal error 🫠",
-    "System might be overloaded 😴",
-    "Seems like a connection issue 😌",
-    "I'm a little confused right now 🥺",
-    "There was a problem during processing 😵",
-    "I'm not functioning properly at the moment 😤",
-    "Ran into an unexpected error 🫤",
-    "Restarting myself, please wait 😔",
-    "Dealing with some technical difficulties 💗"
+    "Ahh, that Asad always does something naughty to me. Blame him, not me. 😒",
+    "It’s always Asad doing something naughty, never me. Stop blaming me! 😤",
```

Ruff (0.13.3)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)